### DISCUSSION TITLE
Filament v3, Config does not come with 'path' option => env('FILAMENT_PATH', 'admin')

### DISCUSSION THREAD

**Discussion by B-Carcamo** [filamentphp/filament#7494] [Help]
Filament version 3 in the config file generated by 'php artisan vendor:publish --tag=filament-config' does not have the option 'path'=> env('FILAMENT_PATH', 'admin'), to change the path "/admin/login" to "/", is it a bug?

**Comment by danharrin** [comment]
https://filamentphp.com/docs/3.x/panels/configuration#changing-the-path

**Reply by B-Carcamo** [reply]
Thanks...
